src/main/java/org/fundacionjala/coding/cesar/FizzBuzz.java

```diff
@@ -0,0 +1,51 @@
+package org.fundacionjala.coding.cesar;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ *
+ * @author admin-hp
+ */
+public class FizzBuzz {
+    private static final int FIVE = 5;
+    private static final int THREE = 3;
+    /**
+     * @param range List from int.
+     * @return list String replace.
+     */
+    public List<String> doBuzzFizz(final List<Integer> range) {
+
+        List<String> lst = range
+                .stream()
+                .map(i -> (Integer.toString(i)))
+                .collect(Collectors.toList());
+        return lst.stream()
+                .map(i ->
+                        this.isStringNumber(i) == true && Integer.parseInt(i)
+                                % FIVE == 0 & Integer.parseInt(i) % THREE == 0 ? "fizzbuzz" : i
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Correct this "&" to "&&" and extract the right operand to a variable if it should always be evaluated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2178)